lib/test/cdo/test_i18n_string_url_tracker.rb

```diff
@@ -71,6 +71,30 @@ def test_log_given_data_should_call_firehose
     assert_equal(@firehose_record, test_record)
   end
 
+  def test_log_given_url_with_query_string_should_call_firehose_without_query_string
```

Comment:
any reason not to write these as tests specifically of the `normalize_url` functionality? We could have them just call that method directly and assert on the results

Reply:
`normalize_url` is a private method and we only care about how it changes the behavior of `log`. Since 'log' is a public  method, we only care about maintaining its behavior.